src/server_manager/electron_app/test_action.sh

```diff
@@ -14,5 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-do_action server_manager/electron_app/build
+rm -rf $BUILD_DIR/js
```

Comment:
I think we should delete only `$BUILD_DIR/js/server_manager/electron_app`
An action shouldn't interfere with another. I could imagine a root test action that calls all the other test actions

Reply:
If we want to take this out further, we could `export BUILD_DIR=$(mktemp -d)` in our test actions so we can completely isolate.  Maybe that's overkill?

Reply:
I agree with that, the reason I removed the entire test directory is to prevent jasmine from running previously built tests, since we have a single jasmine config.

I now delete each action test directory after (succesfully) running the tests.